advocacy_docs/partner_docs/PreciselyConnectCDC/07-Support.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Support and Logging Details'
+description: 'Details of the Support process and logging information'
+---
+
+## Support
+
+Technical support for the use of these products is provided by both EDB and Precisely.  A proper support contract is required to be in place at both EDB and Precisely.  A support ticket can be opened on either side to start the process.  If it is determined through the support ticket that resources from the other vendor is required, the customer should open a support ticket with that vendor through normal support channels.  This will allow both companies to work together to help the customer as needed.
+
+## Logging
+
+**EDB Postgres Advanced Server Logs:**
+
+Navigate to the `Data` directory in your chosen EDB Postgres Advanced Server instance and from here you can navigate to `log`, `current_logfiles` or you can navigate to the `postgresql.conf` file where you can customize logging options or enable `edb_audit` logs.
+
+**PostgreSQL Logs**
+
+The default log directories for PostgreSQL logs vary depending on the operating system:
+
+- Debian-based system: `/var/log/postgresql/postgresql-x.x.main.log. X.x.`
+
+- Red Hat-based system: `/var/lib/pgsql/data/pg_log`
+
+- Windows: `C:\Program Files\PostgreSQL\9.3\data\pg_log`
+
+**Precisely Logs**
```

Comment:
Are these two different logs or two ways to get to the same logs? either way we should should separate them as I don't think you need to go through steps 1 - 5.  You can do steps 1 -3 or steps 4-5.  

Get logs with Connect CDC Director 
1. Open Connect CDC Director.
2. Click the Tools button.
3. Log File name and location will be displayed under Log File.

Get logs with Connect CDC MonCON
1. Open Connect CDC MonCon.
2. The Connect CDC MonCon log file is located in the Connect CDC program directory in the kernel subdirectory. The name of the file is moncon.log.

Reply:
Updated Guide as per the suggested format.